### PR TITLE
Warning and confirmation prompt when creating projects in unit without 3 unit admin

### PR DESCRIPTION
**Pull request by inaod568**
**Test with:** https://github.com/ScilifelabDataCentre/dds_web/pull/1066

Before submitting a PR to the `dev` branch:
- [x] Tests passing
- [x] Black formatting
- [x] Rebase/merge the `dev` branch
- [ - ] Note in the CHANGELOG

Additional checks before submitting a PR to the `master` branch:
- Change version in `setup.py` (?)

### CODE REVIEW

File: dds_cli/project_creator.py [hunk 1]
```diff
@@ -87,6 +89,24 @@ def create_project(
                 LOG.error(error)
                 return created, created_project_id, user_addition_statuses, error
 
+            warning_message = response.json().get("warning")
+            if warning_message:
+                proceed_creation = rich.prompt.Confirm.ask(
```
Comment (aanil) [inline dds_cli/project_creator.py:94]: This should have a no_prompt option as well.

Comment (inaod568) [reply dds_cli/project_creator.py]: Done!

File: dds_cli/project_creator.py [hunk 2]
```diff
@@ -87,6 +89,32 @@ def create_project(
                 LOG.error(error)
                 return created, created_project_id, user_addition_statuses, error
 
+            warning_message = response.json().get("warning")
+
+            if warning_message:
+                if self.no_prompt:
+                    LOG.info(
```
Comment (inaod568) [inline dds_cli/project_creator.py:96]:
```suggestion
                    LOG.warning(
```

**Review by aanil — APPROVED** [review]
👍 Looks good!

**Pull request merged**